packages/jumpstarter-cli/jumpstarter_cli/j.py

```
@@ -1,6 +1,7 @@
-import concurrent
+import concurrent.futures._base
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)